packages/neon_framework/example/pubspec.lock

```diff
@@ -1666,5 +1666,5 @@ packages:
     source: hosted
     version: "3.1.2"
 sdks:
-  dart: ">=3.4.0 <4.0.0"
+  dart: ">=3.5.0 <4.0.0"
```

Comment:
I think this might cause the analyzer issues? At least it looks weird to me because there should be nothing that is causing this :thinking:

Reply:
I was able to revert the unintended changes to the pubspec.lock, but it didn't fix the problem.

Reply:
I can confirm your findings.
Pinning all sdk versions to 3.4 does not trigger this issue.
The `context.mounted` not working is still an issue that requires us to ignore the errors (I'll report upstream).